generators/csharp/codegen/src/context/AbstractCsharpGeneratorContext.ts

```diff
@@ -124,6 +124,10 @@ export abstract class AbstractCsharpGeneratorContext<
         return this.ir.idempotencyHeaders;
     }
 
+    public shouldGenerateDiscriminatedUnions(): boolean {
+        return this.customConfig["use-discriminated-unions"] ?? false;
+    }
```

Comment:
This is included purely for backwards compatibility, right? I imagine we'll default this to `true` in the next major version release?

Reply:
Correct